src/servers/GatewayServer/gatewayserver.threaded.ts

```diff
@@ -29,7 +29,10 @@ export class GatewayServerThreaded extends EventEmitter {
   appDataChannel: MessageChannel = new MessageChannel();
   loginChannel: MessageChannel = new MessageChannel();
   disconnectChannel: MessageChannel = new MessageChannel();
+  clientInfoChannel: MessageChannel = new MessageChannel();
   internalChannel: MessageChannel = new MessageChannel();
+  reqCount: number = 0;
+  requestQueue: Map<number, any> = new Map();
```

Comment:
this map is never cleared, memleak right here

```suggestion
  requestMap: Map<number, any> = new Map();
```